UnitedIncome/components
# Exporting "Pure" versions of the illustration components

### Description
> Please provide a brief description of your changes below.

Related issue: https://github.com/storybookjs/storybook/issues/9586

Exports a "Pure" version of each illustration to restore the prop table. I ended up doing `import {PureComp as Comp} from './Comp'; in the `.mdx` files just to make it easier to fix back when/if this issue gets resolved from storybooks side. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Run `yarn start`.
* Go to each illustration and look at the prop table.

### Comments
> Any additional comments that will aid with the review process.

N/A
